src/screens/Welcome.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  Text,
+  Image,
+  StyleSheet,
+  View,
+  TouchableOpacity,
+} from 'react-native';
+import {Colors} from '../utils/Colors';
+
+const welcome = ({navigation}) => {
+  return (
+    <SafeAreaView style={styles.welcomeScreenContainer}>
+      <View style={styles.imageContainer}>
+        <Image
+          source={require('../utils/example-covid.png')}
+          style={styles.imageLogo}
+        />
+      </View>
+      <View style={styles.elementsContainer}>
+        <Text style={styles.titleText}>Covid Tracker</Text>
+        {/* <Button
+        title="go privacy"
+        onPress={() => {
+          navigation.navigate('PrivacyPolicy');
+        }}
+      /> */}
+        <View style={styles.btnsContainer}>
+          <TouchableOpacity style={styles.button}>
+            <Text
+              style={styles.buttonText}
+              onPress={() => {
+                navigation.navigate('PrivacyPolicy');
+              }}>
+              Registro
+            </Text>
+          </TouchableOpacity>
+          <TouchableOpacity style={styles.button}>
+            <Text style={styles.buttonText}>Iniciar Sesión</Text>
+          </TouchableOpacity>
+        </View>
+      </View>
+    </SafeAreaView>
+  );
+};
+
+export default welcome;
+
+const styles = StyleSheet.create({
+  welcomeScreenContainer: {
+    height: '100%',
```

Comment:
height is not necessary because you are using `flex: 1` and that property will give you all the space on the screen